test/milton/MiltonShouldClosePosition.t.sol

```diff
@@ -0,0 +1,4797 @@
+            address(_daiMockedToken),
+            address(iporOracle),
+            address(miltonStorageDai),
+            address(_miltonSpreadModel),
+            address(stanleyDai)
+        );
+        MockCase0JosephDai mockCase0JosephDai = getMockCase0JosephDai(
+            address(_daiMockedToken),
+            address(_ipTokenDai),
+            address(mockCase3MiltonDai),
+            address(miltonStorageDai),
+            address(stanleyDai)
+        );
+        prepareApproveForUsersDai(_users, _daiMockedToken, address(mockCase0JosephDai), address(mockCase3MiltonDai));
+        prepareMilton(mockCase3MiltonDai, address(mockCase0JosephDai), address(stanleyDai));
+        prepareJoseph(mockCase0JosephDai);
+        prepareIpToken(_ipTokenDai, address(mockCase0JosephDai));
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(_daiMockedToken), TestConstants.PERCENTAGE_3_18DEC, block.timestamp);
+        vm.prank(_liquidityProvider);
+        mockCase0JosephDai.itfProvideLiquidity(TestConstants.USD_1_000_000_18DEC, block.timestamp);
+        // when
+        uint256[] memory payFixedSwapIds = new uint256[](5);
+        uint256[] memory receiveFixedSwapIds = new uint256[](5);
+        for (uint256 i = 0; i < 5; ++i) {
+            openSwapPayFixed(
+                _userTwo,
+                block.timestamp,
+                TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+                TestConstants.PERCENTAGE_6_18DEC,
+                TestConstants.LEVERAGE_18DEC,
+                mockCase3MiltonDai
+            );
+            payFixedSwapIds[i] = i + 1;
+        }
+        for (uint256 i = 0; i < 5; ++i) {
+            openSwapReceiveFixed(
+                _userTwo,
+                block.timestamp,
+                TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+                TestConstants.PERCENTAGE_1_18DEC,
+                TestConstants.LEVERAGE_18DEC,
+                mockCase3MiltonDai
+            );
+            receiveFixedSwapIds[i] = i + 1;
+        }
+        vm.startPrank(_userTwo);
+        mockCase3MiltonDai.itfCloseSwapPayFixed(3, block.timestamp + TestConstants.PERIOD_28_DAYS_IN_SECONDS);
+        mockCase3MiltonDai.itfCloseSwapReceiveFixed(8, block.timestamp + TestConstants.PERIOD_28_DAYS_IN_SECONDS);
+        mockCase3MiltonDai.itfCloseSwaps(
+            payFixedSwapIds, receiveFixedSwapIds, block.timestamp + TestConstants.PERIOD_28_DAYS_IN_SECONDS
+        );
+        vm.stopPrank();
+        // then
+        for (uint256 i = 0; i < 5; ++i) {
+            IporTypes.IporSwapMemory memory payFixedSwap = miltonStorageDai.getSwapPayFixed(i + 1);
+            assertEq(payFixedSwap.state, TestConstants.ZERO);
+        }
+        for (uint256 i = 0; i < 5; ++i) {
+            IporTypes.IporSwapMemory memory receiveFixedSwap = miltonStorageDai.getSwapReceiveFixed(i + 1);
+            assertEq(receiveFixedSwap.state, TestConstants.ZERO);
+        }
```

Comment:
Add additional assertion in this test which was used in old test ""should close 5 pay fixed, 5 receive fixed positions in one transaction - case 3, some of them are already closed, call static"